sql/telemetry_derived/clients_daily_v7.sql

```diff
@@ -114,201 +131,179 @@ WITH
+  SUM(scalar_parent_devtools_accessibility_node_inspected_count) AS scalar_parent_devtools_accessibility_node_inspected_count_sum,
+  SUM(scalar_parent_devtools_accessibility_opened_count) AS scalar_parent_devtools_accessibility_opened_count_sum,
+  SUM(scalar_parent_devtools_accessibility_picker_used_count) AS scalar_parent_devtools_accessibility_picker_used_count_sum,
+  udf_aggregate_map_sum(ARRAY_AGG(scalar_parent_devtools_accessibility_select_accessible_for_node)) AS scalar_parent_devtools_accessibility_select_accessible_for_node_sum,
+  SUM(scalar_parent_devtools_accessibility_service_enabled_count) AS scalar_parent_devtools_accessibility_service_enabled_count_sum,
+  SUM(scalar_parent_devtools_copy_full_css_selector_opened) AS scalar_parent_devtools_copy_full_css_selector_opened_sum,
+  SUM(scalar_parent_devtools_copy_unique_css_selector_opened) AS scalar_parent_devtools_copy_unique_css_selector_opened_sum,
+  SUM(scalar_parent_devtools_toolbar_eyedropper_opened) AS scalar_parent_devtools_toolbar_eyedropper_opened_sum,
+  NULL AS scalar_parent_dom_contentprocess_troubled_due_to_memory_sum, -- deprecated
+  SUM(scalar_parent_navigator_storage_estimate_count) AS scalar_parent_navigator_storage_estimate_count_sum,
+  SUM(scalar_parent_navigator_storage_persist_count) AS scalar_parent_navigator_storage_persist_count_sum,
+  SUM(scalar_parent_storage_sync_api_usage_extensions_using) AS scalar_parent_storage_sync_api_usage_extensions_using_sum,
+  udf_mode_last(ARRAY_AGG(search_cohort)) AS search_cohort,
+  udf_aggregate_search_counts(ARRAY_AGG(search_counts)).*,
+  AVG(session_restored) AS session_restored_mean,
+  COUNTIF(subsession_counter = 1) AS sessions_started_on_this_day,
+  SUM(shutdown_kill) AS shutdown_kill_sum,
+  SUM(subsession_length/NUMERIC '3600') AS subsession_hours_sum,
+  SUM(ssl_handshake_result_failure) AS ssl_handshake_result_failure_sum,
+  SUM(ssl_handshake_result_success) AS ssl_handshake_result_success_sum,
+  udf_mode_last(ARRAY_AGG(sync_configured)) AS sync_configured,
+  AVG(sync_count_desktop) AS sync_count_desktop_mean,
+  AVG(sync_count_mobile) AS sync_count_mobile_mean,
+  SUM(sync_count_desktop) AS sync_count_desktop_sum,
+  SUM(sync_count_mobile) AS sync_count_mobile_sum,
+  udf_mode_last(ARRAY_AGG(telemetry_enabled)) AS telemetry_enabled,
+  udf_mode_last(ARRAY_AGG(timezone_offset)) AS timezone_offset,
+  CAST(NULL AS NUMERIC) AS total_hours_sum,
+  udf_mode_last(ARRAY_AGG(update_auto_download)) AS update_auto_download,
+  udf_mode_last(ARRAY_AGG(update_channel)) AS update_channel,
+  udf_mode_last(ARRAY_AGG(update_enabled)) AS update_enabled,
+  udf_mode_last(ARRAY_AGG(vendor)) AS vendor,
+  SUM(web_notification_shown) AS web_notification_shown_sum,
+  udf_mode_last(ARRAY_AGG(windows_build_number)) AS windows_build_number,
+  udf_mode_last(ARRAY_AGG(windows_ubr)) AS windows_ubr
 FROM
-  windowed
-WHERE
-  _n = 1
+  deduplicated
+GROUP BY
+  client_id,
```

Comment:
I had to refactor this to a `GROUP BY` to get away from `Resources exceeded during query execution: Not enough resources for query planning - too many subqueries or query is too complex.`